### ISSUE TITLE
Comments overflow on public solution page

### ISSUE THREAD

**Issue by budmc29** [exercism/exercism#4225]
https://exercism.io/tracks/ruby/exercises/raindrops/solutions/9c1c685842c44cb3bee9e2d19a3acbf5

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/7063792/44006747-8fa92da8-9e81-11e8-9f25-75ffe3e8f340.png">

**Comment by iHiD** [comment]
@jackhughesweb Maybe make the comment box scroll if it overflows?

**Comment by jackhughesweb** [comment]
@iHiD I've added scrolling locally and the comment box appears as following. 
However, I've encountered an issue where a line may not obviously appear to have more content (see the first line) unless the user scrolls, and long paragraphs now appear on a single scrollable line. I'm not 100% sure how to best solve this issue.

![screen shot 2018-08-13 at 19 09 36](https://user-images.githubusercontent.com/1905799/44049611-7d533f4e-9f2c-11e8-9506-a8ce1840c5e8.png)
